In [3]:
from readlif.reader import LifFile

path = f'Microscopy Data Huntingtin aggregates/Raw data #1/20240131_CKR_E10B_mHtt-HAQ25Doxy96H_HA-star580_CCT1-star635P_A11-star460L_nucspotlive488_1.lif'

# .lif dosyasını aç
lif_file = LifFile(path)

# İçindeki tüm görüntüleri listele
for image in lif_file.get_iter_image():
    dims = image.info['dims']  # Boyut bilgilerini al
    print(f"Adı: {image.name}, Boyutlar: {dims}")

Adı: Series001, Boyutlar: Dims(x=2168, y=2168, z=9, t=1, m=1)


- x = 2168, y = 2168 → 2168x2168 piksel boyutunda
- z = 9 → 9 farklı Z-katmanı (derinlik seviyesi, 3D veri olabilir)
- t = 1 → Zaman boyutu 1 (yani zaman serisi içermiyor)
- m = 1 → 1 kanal var (muhtemelen gri ölçekli mikroskop görüntüsü)